### ISSUE TITLE
Any documentation

### ISSUE THREAD

**Issue by isaachinman** [closeio/sync-engine#907]
Hi Close.io team – having a look at this fork.

If I understand correctly, the original Nylas repo was abandoned 7 years ago, and the [docs URL](https://developer.nylas.com/) surely is no longer valid.

I also was not able to find any Swagger or OpenAPI info. Does any documentation exist, besides going digging through source code?

**Comment by squeaky-pl** [comment]
You are right. It was abandoned and we took over. We are maintaining some parts of the original sync-engine we use for [Close CRM](https://www.close.com/), mostly:

- email syncing over IMAP (not sending over SMTP)
- Calendar syncing (just reads, no writes)

Other parts might be working, but since we are not actually using them your mileage will vary.

About API docs, we also use only a subset of the APIs from the day this was forked. We also added some, and changed some of them. At some point I would like to add some documentation, at least for the endpoints we use, but can't promise anything right now.

For now the best documentation might be the tests suite.
